dcp-0007/dcp-0007.mediawiki

```diff
@@ -0,0 +1,193 @@
+<pre>
+DCP: 0007
+Title: Revert Treasury Expenditure Policy
+Author: Matheus Degiovani <contact@matheusd.com>
+Status: Active
+Created: 2021-06-23
+License: CC0-1.0
+License-Code: ISC
+Requires: DCP0006
+</pre>
+
+==Abstract==
+
+This proposes reverting the currently active maximum expenditure policy of the
+treasury account to the one specified in the original Politeia proposal for the
+decentralized treasury.
```

Comment:
If you're "reverting" to the policy approved by stakeholders, isn't it just a bug fix? 

If a developer makes a mistake, which is arguably what has happened here (even if it was a collective decision of many developers), does there need to be a vote to fix it?

Reply:
Yes.  Without a vote, nodes would be hard forked from the network and havoc would ensue.  Distributed consensus requires very special care.